ext/src/zpages/tracez_http_server.cc

```diff
@@ -0,0 +1,96 @@
+#include "opentelemetry/ext/zpages/tracez_http_server.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext {
+namespace zpages {
+
+  void TracezHttpServer::UpdateAggregations() {
+    aggregated_data_ = data_aggregator_->GetAggregatedTracezData();
+  }
+
+  json TracezHttpServer::GetAggregations() {
+    UpdateAggregations();
+    auto counts = json::array();
+
+    for(const auto &aggregation_group: aggregated_data_){
+      const auto &buckets = aggregation_group.second;
+      const auto &complete_ok_counts = buckets.completed_span_count_per_latency_bucket;
+
+      auto latency_counts = json::array();
+      for (int i = 0; i < 9; i++) {
```

Comment:
Nit: please add a comment why 9.

Reply:
I changed this loop in my PR, It loops over latency boundaries instead

Reply:
```
for (auto boundary = LatencyBoundary::k0MicroTo10Micro;
        latency_counts.push_back(complete_ok_counts[i]);	       boundary != LatencyBoundary::k100SecondToMax; ++boundary) {
        latency_counts.push_back(complete_ok_counts[boundary]);
      }	      }
```
That's what it looks like